common/cmake/install.cmake

```diff
@@ -0,0 +1,72 @@
+function(INSTALL_DEVICE_FILES)
+  # ~~~
+  # INSTALL_DEVICE_FILES(
+  #   DEVICE <device>
+  #   PACKAGE <package>
+  #   DEVICE_TYPE <device_type>
+  #   )
+  # ~~~
+  #
+  set(options)
+  set(oneValueArgs DEVICE PACKAGE DEVICE_TYPE)
+  set(multiValueArgs)
+  cmake_parse_arguments(
+    INSTALL_DEVICE_FILES
+    "${options}"
+    "${oneValueArgs}"
+    "${multiValueArgs}"
+    ${ARGN}
+  )
+
+  set(DEVICE ${INSTALL_DEVICE_FILES_DEVICE})
+  set(DEVICE_TYPE ${INSTALL_DEVICE_FILES_DEVICE_TYPE})
+  set(PACKAGE ${INSTALL_DEVICE_FILES_PACKAGE})
+
+  get_target_property(USE_ROI ${DEVICE_TYPE} USE_ROI)
+  if(USE_ROI OR USE_ROI STREQUAL "USE_ROI-NOTFOUND")
+    message(STATUS "Skipping device files installation for ${DEVICE}-${PACKAGE} type: ${DEVICE_TYPE}")
+    return()
+  endif()
+
+  set(INSTALL_FILES)
+
+  # Get files to be installed
+  get_target_property_required(HAS_LOOKAHEAD ${DEVICE} "${PACKAGE}_HAS_LOOKAHEAD_CACHE")
+  get_target_property_required(HAS_PLACE_DELAY ${DEVICE} "${PACKAGE}_HAS_PLACE_DELAY_CACHE")
+  if(${HAS_LOOKAHEAD})
+    get_target_property(LOOKAHEAD_FILE ${DEVICE} ${PACKAGE}_LOOKAHEAD_FILE)
+    list(APPEND INSTALL_FILES ${LOOKAHEAD_FILE})
+  endif()
+  if(${HAS_PLACE_DELAY})
+    get_target_property(PLACE_DELAY_FILE ${DEVICE} ${PACKAGE}_PLACE_DELAY_FILE)
+    list(APPEND INSTALL_FILES ${PLACE_DELAY_FILE})
+  endif()
+  get_target_property_required(RR_GRAPH_FILE ${DEVICE} ${PACKAGE}_OUT_RRXML_REAL)
```

Comment:
This should probably be `${PACKAGE}_OUT_RRBIN_REAL` now
https://github.com/SymbiFlow/symbiflow-arch-defs/blob/master/common/cmake/devices.cmake#L661